Tests/scripts/hook_validations/integration.py

```diff
@@ -181,6 +187,53 @@ def is_changed_subtype(self):
 
         return self._is_valid
 
+    def is_valid_beta(self, is_new=False):
+        """Validate that that beta integration has correct beta attributes"""
+
+        if not all([self._is_display_contains_beta(), self._has_beta_param()]):
+            self._is_valid = False
+        if is_new:
+            if not all([self._id_has_no_beta_substring(), self._name_has_no_beta_substring()]):
```

Comment:
why is this only for new ones?

Reply:
@ronykoz 
some of the old contains beta in the name/id and we don't want to change them

Reply:
@yaakovi is correct. we already discussed this and decided that new beta integrations should not contain the substring "beta" in the id an name fields. but we will not change existing ones.